spi/src/main/java/com/flipkart/grayskull/spi/annotations/ReadOnly.java

```diff
@@ -0,0 +1,30 @@
+package com.flipkart.grayskull.spi.annotations;
```

Comment:
why is this in spi?

Reply:
Also, don't you need the "bypass" annontation rather than the readonly. That's basically the GET calls.. Now if we want some non-GET API to be available during read only context, we can add that annotation there. Anyone writing their own app can't modify existing grayskull APIs anyways